changelogs/unreleased/5880-anshulahuja98

```diff
@@ -0,0 +1 @@
+Add Proposal to add support for JSON Substitutions through RestoreItemAction plugin.
```

Comment:
Designs do not affect release content so I do not think changelogs applies here.